.github/workflows/ci.yaml

```diff
@@ -139,12 +126,57 @@ jobs:
           only-new-issues: true
           version: v1.42
 
+  chart-lint:
+    name: Helm chart Lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: helm/chart-testing-action@v2.1.0
```

Comment:
They released v2.2.1, it can make sense to update

Reply:
Solved in [#bb085fd](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/40/commits/bb085fdae4dd77d656cf0a01cf238efca5f133e3)